go/consensus/api/api.go

```diff
@@ -204,6 +204,9 @@ type Vote struct {
 
 // Status is the current status overview.
 type Status struct { // nolint: maligned
+	// Status is an concise status of the consensus backend.
+	Status string `json:"string"`
```

Comment:
Probably should make its own type with some constants instead of this being directly a string.

Reply:
Added